app/src/App/App.test.tsx

```diff
@@ -1,6 +1,22 @@
 import {render, screen} from '@testing-library/react';
 import App from './App';
 
+test('Checks route paths are correct', () => {
```

Comment:
Can you also create another test that checks that the order of the routes are correct as well? 

I think this test will pass still if the routes are in this order: default -> cse100s -> cse300s -> cse500s -> cse400s

Reply:
I added a test, but I wonder if testing the order of routes is needed, since they're not visible elements. The routes could be defined in any order and still work as URLs. Unless you meant something different?

Reply:
In the test, it looks like it just grabs all the routes and checks if its within our expectations which is good.

However, if the CSE 100s tab took you to CSE 400s, this test wouldn't capture that (I think). So another test that checks that the correct tab takes you to the correct route should be here.

Reply:
App doesn't handle navigation itself, just defines what routes exist and their contents, so I added the test to Homepage.test because the is the Tab's onclick function that handles the actual routing.